notebooks/TestPhasePlaneExport.ipynb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
We have another `PhasePlane.ipynb` notebook here.
Would it be worthy for these two to be merged ?
Also maybe better to do a "clean all cell output" before we comit notebooks, to avoid having local paths in them ?